expected/plv8_2.out

```diff
@@ -0,0 +1,740 @@
+-- CREATE FUNCTION
```

Comment:
what are you trying to add here?

Reply:
Sorry; it's explained in the commit message, but this UI doesn't show that unless you go looking for it.

One of the tests in plv8.sql -- `text_to_int4('abc')` -- fails under PG12 because the error message changed from `invalid input syntax for integer:` to `invalid input syntax for type integer:`.

I was trying to make the tests pass on all versions with minimal re-writing, so I added plv8_2.out as an alternative expected/ file that uses the new error message. I'd be happy to change it if you'd prefer split this case into a separate test file, or something like that.

Reply:
I think I'd prefer a second PR, separate out the concerns.

Reply:
No problem; I put them together here because the tests won't pass against PG12 without the fcinfo change. Which speaks well for the test suite.

Reply:
updating `.travis.yml` should also go in the other PR :)